ntbs-service-unit-tests/Services/NotificationServiceTest.cs

```diff
@@ -289,6 +289,22 @@ public async Task UpdateImmunosuppressionDetails_StripsOtherDescriptionWhenHasOt
             VerifyUpdateDatabaseCalled();
         }
 
+        [Fact]
+        public async Task UpdatePatientDetails_FormatsNhsNumber()
+        {
+            var reference = new PatientDetails { NhsNumber = "12345" };
+            var input = new PatientDetails { NhsNumber = "12 345" };
+            var notification = new Notification();
+
+            await _notificationService.UpdatePatientDetailsAsync(notification, input);
+
+            Assert.Equal(reference.NhsNumber, input.NhsNumber);
+
+            _mockContext.Verify(context => context.SetValues(notification.PatientDetails, input));
+            _mockAlertService.Verify(x => x.AutoDismissAlertAsync<DataQualityBirthCountryAlert>(It.IsAny<Notification>()), Times.Once);
+            VerifyUpdateDatabaseCalled();

```

Comment:
Super minor but we usually add `Arrange/Assert/Act` comments to our tests, could you add them here?